src/OlStyleParser.ts

```diff
@@ -1238,7 +1239,7 @@ export class OlStyleParser implements StyleParser<OlStyleLike> {
 
     const baseProps: OlStyleIconOptions = {
       src: isSprite(symbolizer.image) ? symbolizer.image.source as string : symbolizer.image as string,
-      crossOrigin: 'anonymous',
+      crossOrigin: null,
```

Comment:
Can you elaborate on why this is needed?

Reply:
I actually don't know why this was set to "anonymous". The default value seems to be"null" when creating a minimum image style:
https://github.com/geostyler/geostyler-openlayers-parser/pull/827/files#diff-0c5b55a3fe5d9526721957d60430942cb5df5c522dbe670f1c6bded92110bad7R10-R14

Reply:
This is include to ensure that the canvas is not tainted when requesting images from a foreign origin. I think it is better to keep it in.